### PR TITLE
[Owners] [Tiny] Removing extra space in mako file

### PR DESCRIPTION
**Pull request by kruthi-NI**
### What does this Pull Request accomplish?

In one of the previous PR, there was an extra space added to mako file by mistake. This PR removes that extra space.

Note: In that previous PR, only change to mako file was submitted but the effect it had on some of the generated files were not submitted. The confusion happened because in visual studio, it was not showing the diff correctly for that extra space in generated files and hence was missed.

### Why should this Pull Request be merged?

To fix indentation issues, this PR should be submitted.

### What testing has been done?

After this change, generated files during the local build do not have that extra space.

### CODE REVIEW

**Review by doshirohan — APPROVED** [review]
Add [Tiny] in PR title.

**Review by ni-pkhare — APPROVED** [review]

**Review by jonlugoatx — APPROVED** [review]

**Review by reckenro — COMMENTED** [review]
Looks good. Don't think it lets me approve after it's been merged in.

**Pull request merged**
